Tutorials/Your scientific publication A-Z/1. Introduction/README.md

```diff
@@ -1,3 +1,20 @@
 # Introduction
 
-In the end of this tutorial, the user will have.....
+In the end of this tutorial, the user will have and idea about the workflow areas in INSERI.
```

Comment:
will there be one main tutorial or several ones, I don't know what is better, what do you think?

Reply:
ok, read the following part, never mind